en/peripherals/remote_id.md

```diff
@@ -8,43 +10,46 @@ Remote ID is a government mandated technology for UAVs in Japan, the United Stat
 The specification requires that UAVs broadcast data such as: real-time location/altitude, serial number, operator ID/location, status, etc.
 
 PX4 works with Remote ID modules that target the FAA [standard Remote ID rules](https://www.faa.gov/uas/getting_started/remote_id).
-These are designed to be integrated into the vehicle, and broadcast Open Drone ID messages using id, position, and other information that is supplied by an autopilot.
+These are designed to be integrated into the vehicle, and broadcast Open Drone ID messages (Open Drone ID is an open source implementation of Remote ID) using id, position, and other information that is supplied by an autopilot.
 The "standard rules" modules enable less restrictive operation than "broadcast rules" modules, which are standalone modules with an integrated GPS that do not have any communication with the autopilot.
 
 ## Supported Hardware
 
-PX4 integrates with Remote ID hardware that supports the [Open Drone ID](https://mavlink.io/en/services/opendroneid.html) MAVLink protocol (Open Drone ID is an open source implementation of Remote ID).
+PX4 integrates with Remote ID hardware that supports:
+
+- [Open Drone ID](https://mavlink.io/en/services/opendroneid.html) MAVLink protocol<Badge type="tip" text="PX4 v1.14" />
+- Remote ID over CAN<Badge type="tip" text="PX4 main (v1.16)" />
+
+  ::: warning
```

Comment:
FYI I moved your warning down here from the top to be a little less confronting - to easy to misread as all remote ids not supported except in main.

Reply:
This is from the original Remote ID Documentation that @julianoes wrote. I just pulled in those changes so that I didn't write over or rewrite any of the information that was already present.

Reply:
:-) All good!